docs/azure_arc_jumpstart/azure_arc_k8s/eks/eks_terraform/_index.md

```diff
@@ -35,39 +30,25 @@ The following README will guide you on how to use the provided [Terraform](https
 
 * [Create a free Amazon Web Service's account](https://aws.amazon.com/free/)
 
-* [Install Helm 3](https://helm.sh/docs/intro/install/)
+* [Install Terraform >=1.0](https://learn.hashicorp.com/terraform/getting-started/install.html)
 
-* [Install Terraform >=0.12](https://learn.hashicorp.com/terraform/getting-started/install.html)
+* [Install Kubectl](https://kubernetes.io/docs/tasks/tools/)
 
-* Create Azure service principal (SP)
+* Login to Azure CLI
 
-    To be able to complete the scenario and its related automation, Azure service principal assigned with the “Contributor” role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
+    To be able to complete the scenario and its related automation, you will need access to an Azure subscription in which you are assigned the role of at least "Contributor".
 
     ```shell
     az login
-    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    az account set --subscription "<SubscriptionID>"
     ```
 
     For example:
 
     ```shell
-    az ad sp create-for-rbac -n "http://AzureArcK8s" --role contributor
+    az account set --subscription "d4617bb1-551c-4450-88a7-f3e4d17cbe46"
     ```
 
-    Output should look like this:
-
-    ```json
-    {
-    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
-    "displayName": "AzureArcK8s",
-    "name": "http://AzureArcK8s",
-    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
-    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-    }
-    ```
-
-    > **Note: The Jumpstart scenarios are designed with as much ease of use in-mind and adhering to security-related best practices whenever possible. It is optional but highly recommended to scope the service principal to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/en-us/azure/role-based-access-control/best-practices)**
-
```

Comment:
why remove the instructions on how to create SP and the output?

Reply:
In this scenario, the Service Principal isn't used at all. As it was originally designed, the user was instructed to create an SP, login to the Azure CLI as the SP, then onboard the K8s cluster. To me, this is incredibly roundabout because in real life there is no reason to login as a Service Principal to onboard a single K8s cluster. If you really want the instructions included an have the user wondering "wait, why did I create a Service Principal" for the rest of the scenario, then we can absolutely put this section back in......but in my opinion, Service Principal scenarios are more centered around automation where a user needs to onboard more than one K8s cluster, or at a minimum needs to have another process onboard the cluster on their behalf.